app/src/main/java/com/otaman/weather/ui/MainScreen.kt

```diff
@@ -0,0 +1,302 @@
+package com.otaman.weather.ui
+
+import androidx.compose.foundation.*
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.*
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.KeyboardArrowDown
+import androidx.compose.material.icons.filled.KeyboardArrowRight
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.otaman.weather.R
+import com.otaman.weather.ui.theme.BlueDark
+import com.otaman.weather.ui.theme.BlueLight
+import com.otaman.weather.ui.theme.WeatherTheme
+
+@Composable
+fun AppBar() {
```

Comment:
I would suggest to use `private` visibility modifier unless a function must be `public`. The same goes for all functions in this file (including previews)